OpenDreamClient/Rendering/DreamIcon.cs

```diff
@@ -186,7 +186,7 @@ sealed class DreamIcon {
 
             AtlasTexture frame = CurrentFrame;
             if (frame != null) {
-                handle.DrawTexture(frame, position, Appearance.Color);
+                handle.DrawTexture(frame, position, Appearance.Color); // TODO: Does not consider ColorMatrix
```

Comment:
I'm leaving this as a TODO since getting a handle to the cached shader from within the DreamIcon would require some IoC resolution memes I don't really want to deal with.